qw5q_gold.yml

```diff
@@ -9,35 +9,126 @@ topology:
 
 instruments:
     twpa_pump:
-        frequency: 6_535_900_000
-        power: 4
+        frequency: 6545500000
+        power: 1.4
+
+    qcm_bb0:
+        o1:
+            channel: L4-5
```

Comment:
As I wrote in the qibolab PR I have not reviewed the new approach in detail yet, however one suggestion would be to avoid putting the channel here. I understand it may be convenient, however this information already exists in the platform `create` method and we should avoid duplication as much as possible, mostly because it can lead to errors (what if you put a different channel here than in the platform creation?). 
Also, the general idea is that `create` should contain static parameters (channel map, qubit names, instrument names and IPs) which do not change during calibration, while the yaml contains the parameters that may change. Channels do not change (unless you plug/unplug cables) so they should be in `create`.

Reply:
Thanks for the fast feedback @stavros11, I was planning to do this modification since it is a static and redundant parameter, but for the moment I just reproduced exactly the parameter in the previous approach.
I will delete it and I will also implement the dumping in the runcard.